docs-source/spring/content/development/cli/_index.md

```diff
@@ -25,19 +23,80 @@ Table of Contents:
   * [List - show details about deployments](#list)
   * [Manage config server data](#config)
   * [GraalVM compile commands](#compile)
+  * [User Management](#user-management)  
+* [Logging Information](#logging)
+
+## Installation
+
+1. CLI is available for Linux on x64 and AArch64, macOS on x64 and AArch64, and Windows x64 architectures. Choose your operating system and proceed to the installation steps for your specific platform:
+
+   * [Linux x86](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-linux-x86)
+   * [Linux AArch64](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-linux-aarch64)
+   * [macOS x86](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-linux-x86)
+   * [macOS AArch64](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-macos-aarch64)
+   * [Windows x86](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-windows-x86)
+
+2. Change to directory where you want to install CLI. The platform-specific binary can be renamed to `oractl` for convenience.
+
+    ```cmd
+    ❯ mv oractl-<version>-<os>-<architecture> /path/to/<obaas>/oractl
+    ```
+  
+    &nbsp;
```

Comment:
There cannot be two code blocks right next to each other without a paragraph in between.